### DISCUSSION TITLE
How to use React Native Maps without an API key?

### DISCUSSION THREAD

**Discussion by putuoka** [react-native-maps/react-native-maps#4794] [Q&A] [answered]
AI said you can use React Native Maps without an API key. 

`You can use Google Maps in React Native without an API key if you only want to render the map. This means that you won't be able to use any of the map's features, such as search, navigation, or geocoding.

To do this, you'll need to use the MapView component from the react-native-maps library.`

I tried to use it, but I was forced to enter an API key. 

<img width="199" alt="image" src="https://github.com/react-native-maps/react-native-maps/assets/20880839/60a8352b-4891-47d3-8159-eb46ae9a4aee">

here's what I did:
1. yarn add react-native-maps
2. edit my playServicesVersion = "18.0.0" in build.gradle
3. yarn android
4. error

**Comment by pTinosq** [comment]
Did you find a solution?

**Reply by putuoka** [reply, marked as answer]
I finally used a webview by implementing an iframe so I could do it without an API key.

**Reply by pTinosq** [reply]
Awesome thanks 👍

**Reply by DaltonHarvey** [reply]
I'm having the same issue, but this one is not working, help

**Reply by josanjohnata** [reply]
Anyway I added this tag because app can not build without this tag. I set value as blank because I will not use API KEY
On the file: `--- android/app/src/main/AndroidManifest.xml`
```
<meta-data 
  android:name="com.google.android.geo.API_KEY"
  android:value="" />
